src/daemon/bitcoind/actions.rs

```diff
@@ -623,9 +623,15 @@ fn poller_main(
     // We use a cache for maintaining our deposits' state up-to-date by polling `listunspent`
     let mut deposits_cache = populate_deposit_cache(&revaultd.read().unwrap())?;
     // When bitcoind is synced, we poll each 30s. On regtest we speed it up for testing.
-    let poll_interval = match revaultd.read().unwrap().bitcoind_config.network {
-        Network::Regtest => Duration::from_secs(3),
-        _ => Duration::from_secs(30),
+    let poll_interval = {
+        let config = &revaultd.read().unwrap().bitcoind_config;
+        match config.poll_interval {
+            Some(i) => i,
+            None => match config.network {
+                Network::Regtest => Duration::from_secs(3),
+                _ => Duration::from_secs(30),
+            },
+        }
```

Comment:
We should remove the special casing for regtest and instead set the `3` seconds in the functional tests config for revaultd (tests/utils.py iirc).

Also, it seems possible with serde [to have a default](https://serde.rs/field-attrs.html#default) directly at de-serialization ? Not a serde expert at all here but it would be neat to have the `30` default in the config file.